core/src/commitment.rs

```diff
@@ -113,15 +112,10 @@ impl AggregateCommitmentService {
                             break;
                         }
 
-                        if let Err(e) = Self::run(&receiver, &block_commitment_cache, &exit_) {
-                            match e {
-                                Error::RecvTimeoutError(RecvTimeoutError::Disconnected) => break,
-                                Error::RecvTimeoutError(RecvTimeoutError::Timeout) => (),
-                                _ => info!(
-                                    "Unexpected error from AggregateCommitmentService: {:?}",
-                                    e
-                                ),
-                            }
+                        if let Err(RecvTimeoutError::Disconnected) =
```

Comment:
Any particular reason you removed the `match` here and in the transaction_status_service, but not in similar constructions below?

Reply:
I only removed the match when the new error enum was so small that there was no longer any unexpected error cases.